test/mocked-actions/cacheHitCollection.js

```diff
@@ -0,0 +1,17 @@
+import { REDUX_WP_API_CACHE_HIT } from '../../src/constants/actions';
```

Comment:
Why is the name of this file in a different standard? Why not `cache-hit-collection.js`?